index.html

```diff
@@ -1395,7 +1395,7 @@ <h4>
     <h2>The <dfn>NDEFRecordType</dfn> enum</h2>
       <p>
         This enum defines the set of known types for a <a>NDEFRecord</a>. The
-        <a href="#data-mapping">data mapping</a> section describes how
+        <a href="#data-mapping"></a> section describes how
```

Comment:
```suggestion
        [[[#data-mapping]]] section describes how
```

Reply:
Odd... what's it say?

Reply:
I may find the root cause, it requires an `id` definition to the corresponding section.

Reply:
Ah, yeah :) Good point. Forgot about that.